circleci/docker-publish

```diff
@@ -43,8 +44,12 @@ image_exists() {
 
 check_ecr_vars() {
   # ECR required env vars
```

Comment:
Aren't these injected by the CircleCI environment?

We could just see if the role has been injected via an environment variable, and then use `aws sts assume-role-with-web-identity`, for minimal changes to `ci-scripts`.

Reply:
I believe that is what the CircleCI orb is doing under the hood. See https://circleci.com/developer/orbs/orb/circleci/aws-cli#commands-assume_role_with_web_identity. Basically, this will make it such that we do not rely on the AWS orb

Reply:
Yes. That's what it does under the hood.

If we move that functionality here (maybe as a util function called into certain `ci-scripts`), then we avoid having to update all of the apps' `.circleci/config.yml`s.